cloudify_cli/cli/cfy.py

```diff
@@ -150,11 +150,18 @@ def recommend(possible_solutions):
             logger.info('  - {0}'.format(solution))
 
     def new_excepthook(tpe, value, tb):
+        traceback.print_exception(
+            etype=tpe,
+            value=value,
+            tb=tb,
+            file=open(DEFAULT_LOG_FILE, 'a'))
```

Comment:
is this someething standard?
shouldnt it be something like
with open(....) as f:
  traceback.print_exception(.... file=f)?